src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPathContextProvider.cs

```diff
@@ -310,5 +312,63 @@ public IVsPathContext GetSolutionPathContext()
                 pathContext.FallbackPackageFolders.Cast<string>(),
                 trie);
         }
+
+        private IList<Project> GetProjectsInSolution(DTE dte)
+        {
+            var supportedProjects = dte.Solution.Projects;
+            Projects projects = dte.Solution.Projects;
+
+            List<Project> projectList = new List<Project>();
```

Comment:
See rule 10 in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style.